api/v1/model/src/commonMain/kotlin/EcosystemStats.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.api.v1.model
+
+import kotlinx.serialization.Serializable
+
+/**
+ * A data class representing statistics for an ecosystem.
+ */
+@Serializable
+data class EcosystemStats(
+    /**
+     * The name of the ecosystem.
+     */
+    val name: String,
+    /**
```

Comment:
Use usually prefer to have empty lines before these to visually group the comment to the property. BTW, short comments can also use the more compact `/** The name of the ecosystem. */` one-line syntax.